apps/spruce/src/pages/task/TaskTabs.tsx

```diff
@@ -171,7 +171,7 @@ export const TaskTabs: React.FC<TaskTabProps> = ({ isDisplayTask, task }) => {
         }
       >
         {/* @ts-expect-error: FIXME. This comment was added by an automated script. */}
```

Comment:
Unrelated but I think the `ts-expect-error` is related to the `TaskTabs` component being typed wrong. 
It thinks `Task` can be undefined meanwhile we guard against it  being undefined [here](https://github.com/evergreen-ci/ui/blob/5593651aecf0b3105f08c6042450cc96ba361ea8/apps/spruce/src/pages/Task.tsx#L155)